global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "3.0.100-preview5-011568",
+    "dotnet": "3.0.100-preview7-012517",
```

Comment:
Preview 6 didn't have the roslyn rename you were looking for. I updated to a preview 7 SDK. This mechanism should allow you to freely update to any of the latest SDK you want. Let me know if this doesn't work.